eslint.config.js

```diff
@@ -0,0 +1,193 @@
+const prettier = require('eslint-config-prettier');
+const globals = require('globals');
+const mochaNoOnly = require('eslint-plugin-mocha-no-only');
+
+module.exports = [
+  prettier,
+  {
+    rules: {
```

Comment:
Should leave a comment [as to why we don't configure our code to allow `globals.browser`](https://github.com/dequelabs/axe-core/blob/develop/.eslintrc.js#L104C9-L104C79)

```js
    // lib files should not access global window properties without going through window (i.e. do not add globals.browser)
```